### DISCUSSION TITLE
Gradle 8.0 broke build of all my sample projects

### DISCUSSION THREAD

**Discussion by hantsy** [micronaut-projects/micronaut-core#8772] [Q&A] [answered]
Failed to notify project evaluation listener.
   > The value for task ':test' property 'testFrameworkProperty' is final and cannot be changed any further.
   > The value for task ':test' property 'testFrameworkProperty' is final and cannot be changed any further.

https://github.com/hantsy/micronaut-sandbox/pull/228

**Comment by melix** [comment, marked as answer]
Dang, I think Gradle re-introduced a change which already broke us months ago: https://github.com/micronaut-projects/micronaut-gradle-plugin/issues/296

I didn't get an answer :(

**Reply by melix** [reply]
I have created a PR on 3.7.x to workaround this issue: https://github.com/micronaut-projects/micronaut-gradle-plugin/pull/640

Another option is to add explicitly to your build scripts that you use `useJUnitPlatform()`, e.g:

```
micronaut {
    runtime("netty")
    testRuntime("junit5")
    processing {
        incremental(true)
        annotations("wazaaaaa.*")
    }
}

tasks.withType(Test).configureEach {
   useJUnitPlatform()
}
```

**Reply by hantsy** [reply]
I think Micronaut should change to align with Gradle 8.0

**Reply by melix** [reply]
You can upgrade to version 3.7.2 of the plugin which fixes this problem.
